client/src/app/site/assignments/services/assignment-poll.service.ts

```diff
@@ -77,19 +81,22 @@ export class AssignmentPollService extends PollService {
     }
 
     public getPercentBase(poll: PollData): number {
-        const base: PercentBase = poll.onehundred_percent_base;
+        const base: AssignmentPollPercentBase = poll.onehundred_percent_base as AssignmentPollPercentBase;
         let totalByBase: number;
         switch (base) {
-            case PercentBase.YN:
+            case AssignmentPollPercentBase.YN:
                 totalByBase = this.sumOptionsYN(poll);
                 break;
-            case PercentBase.YNA:
+            case AssignmentPollPercentBase.YNA:
                 totalByBase = this.sumOptionsYNA(poll);
                 break;
-            case PercentBase.Valid:
+            case AssignmentPollPercentBase.Votes:
```

Comment:
@FinnStutzenstein @emanuelschuetze what should be calculated here? I don't know what "votes" as a poll percent base should calculate